src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -67,6 +74,8 @@ class VAMap extends Component {
 
     window.addEventListener('resize', this.debouncedResize);
 
+    setZoomEvents();
+
```

Comment:
There is a potential race condition here. You are calling `setZoomEvents` in `componentDidMount`, but there is no guarantee that the map will have been fully rendered by then.

If the `fl-map-zoom-in/out` events are really needed (maybe this can be derived from the dataLayer), you might see if there's a callback in the Leaflet API you can leverage instead of calling it here.

Reply:
Or, if there is no callback available, maybe you could do it in `handleBoundsChanged`? That way you know the map has been loaded for sure. You could set a flag in the component state to track whether `setZoomEvents` has been called.

Reply:
I think` onViewportChanged: (viewport: {center: ?[number, number], zoom: ?number}) => void (optional): fired after the viewport changed.` is what I need as center and zoom level update in the callback.  - https://react-leaflet.js.org/docs/en/components